wgpu-native/src/device.rs

```diff
@@ -233,13 +236,17 @@ impl<B: hal::Backend> Device<B> {
             }.unwrap()
         );
 
+        // don't start submission index at zero
```

Comment:
Why isn't this ok?

Reply:
because everything is initialized as 0, so we don't want the first frame to falsely considered completed before it truly finishes